docs/adr/0006-monitoring-structure.md

```diff
@@ -0,0 +1,55 @@
+# Application Monitoring in Operate First using Prometheus
+
+* Status: accepted
+* Deciders: 4n4nd, anishasthana, hemajv, HumairAK, tumido
+* Date: 2021-01-06
+
+Technical Story: [issue-1](https://github.com/operate-first/blueprint/issues/8)
+                 [issue-2](https://github.com/operate-first/apps/issues/57)
+
+## Context and Problem Statement
+
+Since there will be multiple services deployed in the Operate First environment
+(ex. Jupyterhub, Argo, Superset, etc), we need to be able to monitor them. <br>
+To do so we will use [Prometheus](https://prometheus.io/).
+
+How do we structure the Prometheus-Operator/Prometheus Deployments in
+Operate First?
+
+## Decision Drivers <!-- optional -->
+
+* Access to cluster-wide permissions
```

Comment:
What about api conflicts between cluster monitoring/user workload monitoring and the prometheus community operator?